### PR TITLE
run_tasks.py: Python3 don't have exception.message anymore

### PR DESCRIPTION
**Pull request by shyukri**
This was probably working fine due to six library which was removed
recently: https://github.com/ceph/teuthology/commit/9f99b9298265be583c350c71de47109bac4bf6f1

Error seen:

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/worker/src/github.com_ceph_teuthology_master/teuthology/contextutil.py", line 33, in nested
    yield vars
  File "/home/worker/src/github.com_ceph_teuthology_master/teuthology/task/install/__init__.py", line 612, in task
    yield
  File "/home/worker/src/github.com_ceph_teuthology_master/teuthology/run_tasks.py", line 151, in run_tasks
    if e.message == 'too many values to unpack':
AttributeError: 'ValueError' object has no attribute 'message'
```

Signed-off-by: Shyukri Shyukriev <shshyukriev@suse.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
ok to test

**Comment by kshtsk** [comment]
ok to test

File: teuthology/run_tasks.py [hunk 1]
```diff
@@ -255,4 +255,3 @@ def notify_sleep_before_teardown(ctx, stack, sleep_time):
     smtp = smtplib.SMTP('localhost')
     smtp.sendmail(msg['From'], [msg['To']], msg.as_string())
     smtp.quit()
-
```
Comment (kshtsk) [inline teuthology/run_tasks.py:258]: @shyukri this change is not related, could you please undo this?

Comment (shyukri) [reply teuthology/run_tasks.py]: Done. VS Code was misbehaving.

**Comment by kshtsk** [comment]
@susebot run deploy

**Comment by kshtsk** [comment]
@susebot run deploy

**Comment by kshtsk** [comment]
```
2020-07-06T11:39:27.982 INFO:teuthology.orchestra.run.target-pr-1530-046.stdout:Extra Packages for Enterprise Linux 8 - x86_64  8.5  B/s | 4.7 kB     09:32
2020-07-06T11:39:27.984 INFO:teuthology.orchestra.run.target-pr-1530-046.stderr:Failed to download metadata for repo 'epel'
2020-07-06T11:39:28.015 INFO:teuthology.orchestra.run.target-pr-1530-046.stderr:Error: Failed to download metadata for repo 'epel'
2020-07-06T11:39:28.053 DEBUG:teuthology.orchestra.run:got remote process result: 1
```
it is an epel accessibility issue.

Both failures in this and previous runs are unrelated, merging
